test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py

```diff
@@ -240,11 +242,11 @@ def step(context, color: str):
 
 @Then("the heading is \"|any|\"") 
```

Comment:
This can be remove since it is not used and we already have the `the welcome "Tokens" title is present` statement doing the job.